mediathread/templates/dashboard/class_roster.html

```diff
@@ -10,31 +10,42 @@
     <link rel="stylesheet" href="{% static 'js/lib/tablesorter/theme.default.custom.css' %}">
 {% endblock %}
 
+{% block uncompressable_css %}
```

Comment:
Oh okay, i see that uncompressable_js is at the end of the page, while uncompressable_css adds it to `<head>`. I think that's kind of confusing, if we have javascript libraries that need to be in the uncompressable_css block, so might clean this up in the future.